scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -141,6 +168,41 @@ class AssetEditorComponent extends React.PureComponent<IProps> {
                         </FormRow>
                     </FormGroup>
                 )}
+                {this.fieldEnabled('tags') && (
+                    <FormGroup>
+                        <FormRow>
+                            <Autocomplete
+                                key={nextId.name}
```

Comment:
This `key` needs to change with every update, so that the search box's text is cleared after a user selects a Tag to be added